misc/io.github.grangerhub.Tremulous.appdata.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Daniel Rusek <mail@asciiwolf.com> -->
+<component type="desktop-application">
+  <id>io.github.grangerhub.Tremulous</id>
+  <project_license>GPL-2.0</project_license>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <provides>
+    <id>io.github.grangerhub.Tremulous.desktop</id>
+  </provides>
+  <launchable type="desktop-id">io.github.grangerhub.Tremulous.desktop</launchable>
+  <name>Tremulous</name>
+  <developer_name>Dark Legion Development and GrangerHub</developer_name>
+  <summary>Aliens vs Humans, First Person Shooter game with elements of Real Time Strategy (Grangerhub version)</summary>
+  <description>
+    <p>
+      Tremulous is a free, open source game that blends a team based FPS with
+      elements of an RTS.
+    </p>
+    <p>
+      Players can choose from 2 unique races, aliens and humans.
+      Players on both teams are able to build working structures in-game like an
+      RTS. 
+    </p>
+    <p>
+      The GrangerHub version contains the updated game server and
+      client binaries with several stability improvements and new
+      features. See the GrangerHub Github page for more details:
+      https://github.com/GrangerHub/tremulous/
```

Comment:
```suggestion
      features. See the homepage for more details.
```

(Actual URLs with http(s) are not supported inside the description. Also, this URL is already in the homepage tag that gets displayed in AppStream UIs as the project homepage.)

Reply:
Last thing. :)